addons/mail/static/src/models/chat_window_manager.js

```diff
@@ -6,6 +6,7 @@ import { clear, insertAndReplace, link, replace } from '@mail/model/model_field_
 
 const BASE_VISUAL = {
     /**
+     * @deprecated, should use ChatWindowManager/availableVisibleSlots instead
```

Comment:
This PR is for master, why can't we just remove this ?

Reply:
Because we still need it as it's used in `_computeVisual`. And we certainly don't want side-effects in the compute itself, e.g. by manually doing `this.update({ availableVisibleSlots: ... })`

This is a necessity until we break `ChatWindowManager/visual` further, which is not trivial at all, hence the several commits to reach that goal.